web/src/admin/providers/oauth2/OAuth2ProviderForm.ts

```diff
@@ -334,13 +334,14 @@ export class OAuth2ProviderFormPage extends ModelForm<OAuth2Provider, number> {
                         )}
                     >
                     </ak-radio-input>
-                    <ak-switch-input name="includeClaimsInIdToken">
```

Comment:
It's that extra `>` at the end of this line that screwed everything up. It isn't so much "malformed HTML" as "insufficiently specified HTML."